docs/services/slam/cartographer/_index.md

```diff
@@ -149,10 +233,12 @@ Note: This example includes configuration for both the `rplidar` and `cartograph
         "config_params": {
           "mode": "2d"
         },
-        "data_dir": "/Users/my-username/cartographer-directory",
-        "data_rate_msec": 200,
-        "map_rate_sec": 60,
-        "sensors": ["my-rplidar"]
+        "camera": {
+          "name": "my-rplidar",
+          "data_frequency_hz": "5"
+        },
+        "enable_mapping": false,
+        "use_cloud_slam": false
       }
     }
   ],
```

Comment:
Maybe update the versions below, or leave them blank?

Reply:
Good point, I was wondering about that too... it looks like real versions are used in the JSON Template for other modular resources as well (https://docs.viam.com/modular-resources/examples/odrive/, https://docs.viam.com/modular-resources/examples/csi/, https://docs.viam.com/modular-resources/examples/rplidar/). @andf-viam would we want to put a template/placeholder value there instead?

Reply:
Sure, I can update those to placeholder values, makes sense. 👍

Reply:
No worries for this PR, I'll update them all afterwards.

Reply:
@andf-viam sounds great, thank you!